lib/v4/migration-helpers/update-plugin-folder-structure/index.js

```diff
@@ -0,0 +1,83 @@
+'use strict';
+
+const { resolve, join } = require('path');
+const fs = require('fs-extra');
+const chalk = require('chalk');
+
+const { logger } = require('../../../global/utils');
+const convertModelToContentType = require('../convert-models-to-content-types');
+const updateRoutes = require('../update-routes');
+const runJscodeshift = require('../../utils/run-jscodeshift');
+const {
+  createDirectoryIndex,
+  createServerIndex,
+  createContentTypeIndex,
+  moveToServer,
+  moveBootstrapFunction
+} = require('./utils');
+
+const SERVER_DIRECTORIES = ['controllers', 'models', 'middlewares', 'services'];
```

Comment:
Are the policies treated differently?

Reply:
yes because they come from the config folder